lte/gateway/c/core/oai/tasks/nas/esm/esm_data_context.c

```diff
@@ -42,7 +42,7 @@ void free_esm_bearer_context(esm_ebr_context_t* esm_ebr_context) {
       free_traffic_flow_template(&esm_ebr_context->tft);
     }
     if (NAS_TIMER_INACTIVE_ID != esm_ebr_context->timer.id) {
-      esm_ebr_timer_data_t* esm_ebr_timer_data = NULL;
+      esm_ebr_timer_data_t* esm_ebr_timer_data = esm_ebr_context->args;
```

Comment:
This should not be changed. Since on v1.6 nas timers were not migrated. `nas_timer_stop` is supposed to grab the timer arguments and assigned to this one.